tests/main/interfaces-timeserver-control/task.yaml

```diff
@@ -4,6 +4,11 @@ details: |
     This test makes sure that a snap using the timeserver-control interface
     can access timeserver information and update it.
 
+# Debian sid is skipped because "timedatectl set-ntp" fails with the error:
+# "Failed to set ntp: Message recipient disconnected from message bus without replying"
```

Comment:
This is interesting. Thank you for the comment, we can take action on this separately